bindings/pyroot/pythonizations/python/ROOT/pythonization/_rooabspdf.py

```diff
@@ -0,0 +1,66 @@
+# Author: Hinnerk C. Schmidt 02/2021
+
+################################################################################
+# Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+import cppyy
+
+from ROOT import pythonization
+
+
+def __cast(s):
+    # NB: not implemented, returns input
```

Comment:
Personally, I would prefer if the keyword argument in python has a lower case name with underscores, which is standard for Python. So for example `range`, `batch_compute`, etc.

Technically, we could also consider supporting both capitalized and lowercase, but since this interface is new anyway I think we should make a choice.

Reply:
Yes, lower case with underscores is more pythonic. However, one problem I found were translations of `CmdArg`s like `NumCPU`. Should this become `num_CPU` or `num_cpu`? In the latter case one would have to do a lot of manual stitching in order to match and cast these kind of patterns correctly.